CONTRIBUTING.md

```diff
@@ -242,3 +242,28 @@ Below you can find out some hints how to achieve that:
 - Prefer to use methods instead of member functions (def foo: (T) => R)
 - Avoid using AnyVal if the class is in API that will be used from Java
 - Prefer adding the `final` keyword to `case classes` definitions if you have no serious reason to skip it
+
+
+# Examples
```

Comment:
Contributing doc as name point, is for contributors, not for users. IMO it is not a good place to mention about that.

Reply:
Yes, I know. We are going to mention that in the docs obviously. 
Do you think there is any point in informing contributors of this as well?

Reply:
Yes, I know. We are going to mention that in the docs obviously.
Do you think there is any point in informing contributors of this as well?

Reply:
I don't think that informing contributors of this is necessary. They'll find examples in the docs. Does docs will be a part of this PR? e2e perspective would help to check if everything will fit together.

Reply:
No, it won't be :D Documentation changes are a huge part of the epic and there are separate tasks for that.